book/src/Example.md

```diff
@@ -123,42 +250,168 @@ Events:
 
 ## Confirming the setup
 
-Writing inside the app container should be visible in `/tmp` of the `hostpath-driver` container:
+Writing inside the app container should be visible in `/tmp` of the `hostpath` container:
 ```
-$ kubectl exec -ti my-csi-app /bin/sh
+$ kubectl exec -it my-csi-app /bin/sh
 / # touch /data/hello-world
 / # exit
-$ kubectl exec -ti csi-pod -c hostpath-driver /bin/sh
+
+$ kubectl exec -it $(kubectl get pods --selector app=csi-hostpathplugin -o jsonpath='{.items[*].metadata.name}') -c hostpath /bin/sh
 / # find / -name hello-world
-/tmp/aba238a3-21f9-11e8-9164-0242ac110002/hello-world
+/tmp/057485ab-c714-11e8-bb16-000c2967769a/hello-world
+/ # exit
 ```
 
 There should be a `VolumeAttachment` while the app has the volume mounted:
 ```
 $ kubectl get VolumeAttachment
-NAME                                                                   AGE
-csi-7bf49991faecc67f5049ed4b72e56b5935d92fb21b69fe782e74ea26e25863a3   7m
-$ kubectl describe VolumeAttachment
-Name:         csi-7bf49991faecc67f5049ed4b72e56b5935d92fb21b69fe782e74ea26e25863a3
-Namespace:    
+Name:         csi-a4e97f3af2161c6d081b8e96c58ed00c9bf1e1745e89b2545e24505437f015df
+Namespace:
 Labels:       <none>
 Annotations:  <none>
 API Version:  storage.k8s.io/v1beta1
 Kind:         VolumeAttachment
 Metadata:
-  Creation Timestamp:  2018-03-09T20:17:21Z
-  Resource Version:    597
-  Self Link:           /apis/storage.k8s.io/v1beta1/volumeattachments/csi-7bf49991faecc67f5049ed4b72e56b5935d92fb21b69fe782e74ea26e25863a3
-  UID:                 dfaffa08-23d6-11e8-bf78-fcaa1497a416
+  Creation Timestamp:  2018-10-03T13:59:19Z
+  Resource Version:    1730
+  Self Link:           /apis/storage.k8s.io/v1beta1/volumeattachments/csi-a4e97f3af2161c6d081b8e96c58ed00c9bf1e1745e89b2545e24505437f015df
+  UID:                 862d7241-c714-11e8-8911-000c2967769a
 Spec:
   Attacher:   csi-hostpath
   Node Name:  127.0.0.1
   Source:
-    Persistent Volume Name:  kubernetes-dynamic-pv-bb3d8e2a23d611e8
+    Persistent Volume Name:  pvc-0571cc14-c714-11e8-8911-000c2967769a
 Status:
   Attached:  true
 Events:      <none>
 ```
 
-If you encounter any problems, please check the [Troubleshooting page](Troubleshooting.html).
+## Snapshot support
+
+Enable dynamic provisioning of volume snapshot by creating a volume snapshot
+class as follows:
+```
+$ kubectl create -f https://raw.githubusercontent.com/kubernetes-csi/docs/master/book/src/example/csi-snapshotclass.yaml
```

Comment:
Again, I would point to the original repository where the snapshot is defined instead of the file in the example directory.  That way this doc will stay in sync with future changes.